apps/discord-bot/src/constants.ts

```diff
@@ -133,7 +133,7 @@ export const mapBackground = <T extends GamesWithBackgrounds>(
       return ["bedwars", map];
     }
     case ARCADE_MODES:
-      return ["arcade", mode === "seasonal" ? "overall" : mode];
+      return ["arcade", mode === "seasonal" || mode === "pixelParty" ? "overall" : mode];
```

Comment:
```suggestion
      return ["arcade", mode === "seasonal" ? "overall" : mode];
```

Change this because we have backgrounds now.